pkg/registry/servicecatalog/binding/strategy.go

```diff
@@ -93,6 +93,9 @@ func (bindingRESTStrategy) PrepareForCreate(ctx kapi.Context, obj runtime.Object
 	binding.Status = sc.BindingStatus{}
 	// Fill in the first entry set to "creating"?
 	binding.Status.Conditions = []sc.BindingCondition{}
+
+	// TODO: Should we use a more specific string here?
```

Comment:
There's a symbol from k8s we can use - are you cool with doing it in a follow up?  I'd prefer to mutate @krancour's code as little as possible.